Block/Adminhtml/Form/Field/Accounts/BankColumn.php

```diff
@@ -0,0 +1,94 @@
+<?php
+declare(strict_types=1);
+
+namespace Kevin\Payment\Block\Adminhtml\Form\Field\Accounts;
+
+use Magento\Framework\View\Element\Html\Select;
+
+class BankColumn extends Select
```

Comment:
What do you think about `BankColumnViewBuilder` name? `BankColumn` tells me that this is some kind of entity with properties, but i see that it has some logic inside going on. The same rules can be applied for `CountryColumn` as well.

Reply:
renamed